developer-extension/src/panel/components/tabs/infosTab.tsx

```diff
@@ -257,9 +257,12 @@ function formatSessionType(value: string, ...labels: string[]) {
 }
 
 function endSession() {
+  const fourHours = 1000 * 60 * 60 * 4
+  const expires = new Date(Date.now() + fourHours).toUTCString()
+
   evalInWindow(
     `
-      document.cookie = '_dd_s=; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/'
+      document.cookie = '_dd_s=expired=0; expires=${expires}; path=/'
```

Comment:
That was my first approach, however, Benoit [pointed out](https://github.com/DataDog/browser-sdk/pull/2673#discussion_r1553248576) that this would not work for Logs?

Reply:
Yes, valid points!